### DISCUSSION TITLE
Is it the best way to use zlib on msys2 MinGW 64bit

### DISCUSSION THREAD

**Discussion by yuki-kimoto** [msys2/MINGW-packages#8136] [Q&A]
Hi, I'm Yuki Kimoto.

Currently, I try to create Cotton browser application on msys2 MinGW 64bit.

https://github.com/yuki-kimoto/cotton

I want to use libpng to render png image.

but, before that, I understand the way to use zlib becuase libpng depend on zlib

Is it the best way to use zlib on msys2 MinGW 64bit?

zlib static link or zlib.dll?

**Comment by Biswa96** [comment]
I prefer to use dynamic linking i.e. zlib1.dll owned by mingw-w64-x86_64-zlib package.

**Reply by yuki-kimoto** [reply]
@Biswa96 

Thank you.

Oh.

Not only msys2 zlib package, there is zlib package for mingw-w64-x86_64.
